MAUI/DataForm/DataForm-settings.md

```diff
@@ -630,7 +630,7 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-![EditotTextStyle](images/editor-settings/editor-text-style.png)
+![Editor text style for .NET MAUI DataForm.](images/editor-settings/editor-text-style.png)
```

Comment:
Editor text style in .NET MAUI DataForm.

Reply:
updated